scripts/setup-db.sh

```
@@ -0,0 +1,46 @@
+fi
+
+# 3. Git 클론 (DB Compose 실행용)
+REPO_DIR="$HOME/Food-Delivery-Service"
+if ! command -v git &> /dev/null; then sudo apt-get install -y git; fi
+if [ -d "$REPO_DIR" ]; then
+    cd "$REPO_DIR" && git pull origin develop
+else
+    cd "$HOME" && git clone -b develop https://github.com/dPwls0125/Food-Delivery-Service.git
+    cd "$REPO_DIR"
+fi
+fi
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)